cpp/cmake/thirdparty/patches/thrust_override.json

```
@@ -0,0 +1,34 @@
+
+{
+  "packages" : {
+    "Thrust" : {
+      "patches" : [
+        {
+          "file" : "Thrust/install_rules.diff",
+          "issue" : "Thrust 1.X installs incorrect files [https://github.com/NVIDIA/thrust/issues/1790]",
+          "fixed_in" : "2.0"
```

Should these versions be like `X.Y.Z`? Applies several places.
```suggestion
          "fixed_in" : "2.0.0"
```

---

Techincall it doesn't matter as the CMake `VERSION_LESS` check will consider `2`, `2.0`, and `2.0.0` all to be the 'same'. But for consistentcy across rapids-cmake we should use `2.0.0`.